src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --link-color: #00acdc;
+  --header-heihgt: 60px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  padding: 0 50px;
+}
```

Comment:
Don't use tag names for styling (except html and body)
<img width="810" alt="image" src="https://github.com/user-attachments/assets/172df266-ce56-4c68-a773-a17fd240ace6">